common/temp_modules/rush-fabric-website/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "rush-fabric-website",
+  "version": "0.0.0",
+  "private": true,
+  "dependencies": {
+    "@microsoft/load-themed-styles": "^1.2.2",
+    "@microsoft/web-library-build": ">=1.1.0 < 2.0.0",
+    "@types/chai": "^3.4.34",
+    "@types/es6-promise": "0.0.32",
+    "@types/mocha": "^2.2.32",
+    "@types/node": "^6.0.45",
+    "@types/react-addons-test-utils": "^0.14.15",
+    "@types/webpack": "^1.12.35",
+    "es6-promise": "3.2.1",
+    "git-rev": "0.2.1",
+    "gulp": "~3.9.1",
+    "gulp-debug": "2.1.2",
+    "gulp-prompt": "^0.2.0",
+    "gulp-util": "3.0.7",
+    "gutil": "1.6.4",
+    "highlight.js": "9.6.0",
+    "mocha": "~2.5.3",
+    "orchestrator": "^0.3.7",
+    "react-addons-test-utils": "^15.1.0",
+    "react-highlight": "0.8.0",
+    "source-map-loader": "0.1.5",
+    "tslint": "^3.15.1",
+    "tslint-microsoft-contrib": "^2.0.9",
+    "typescript": "^2.0.6",
```

Comment:
updating to appropriate versions of web-library-build inherently has typescript 2.1 (which the main office ui fabric react package is using as well). 

recommend having all packages be on same version of build tools and thus on same version of typescript for compilation.